src/index.mako.html

```diff
@@ -776,6 +776,7 @@
           hrefRegexp: new RegExp('${href_regexp}'),
           cachedApiUrl: apiUrl + cacheAdd,
           cachedAltiUrl: altiUrl + cacheAdd,
+          // TODO to be removed
           imageryMetadataUrl: '//3d.geo.admin.ch/imagery/',
```

Comment:
can you put this in a rc_ variable and use:
DEV : `3d.dev.bgdi.ch `
INT: `terrain100.bgdi.ch`
PROD: `terrain100.geo.admin.ch`

In this way we can test the new varnish version on int without risking of breaking prod...